docs/dependency_support.md

```diff
@@ -0,0 +1,42 @@
+# Dependency support policy
+
+This document outlines our policy for supporting bpftrace's major dependencies.
+Support policy for minor dependencies are done on a case by case basis, usually
+at distro/user request.
+
+## Linux kernel
+
+The linux kernel is bpftrace's biggest runtime dependency. The project's stance
+on kernel support is that for **cross cutting** implementation details (ie.
+something every reasonably sophisticated bpftrace is expected to depend on),
```

Comment:
I'm not sure if I understand what "reasonably sophisticated bpftrace" means

Reply:
Oh, meant to say "bpftrace user". Let's just go with "user"?